travis/travis_install_nightly

```diff
@@ -31,6 +31,12 @@ fi
 ln -s `which nodejs` $HOME/maintainer-quality-tools/travis/node
 npm install -g less less-plugin-clean-css
 
+if [ "${WEBSITE_REPO}" == "1" ]; then
+    rvm install ruby 2.0 &&
+    rvm use 2.0 &&
+    gem install bootstrap-sass &&
```

Comment:
This file starts with `set -e` which will cause it to be aborted on any error, so you don't need those `&&`